wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -357,21 +357,19 @@ Section headings must not skip a heading level (i.e. do not go from a level 2 he
 
 It is possible to redefine a section's identifier, which is used for linking to it directly. Custom identifiers should be used in case the automatically generated ones are too long or contain tricky punctuation marks or images:
 
-<!-- TODO: remove the {id=...} example when https://github.com/ppy/osu-web/issues/8057 is closed -->
-
 ```markdown
 ## My cooldown has passed. How do I appeal? {#appeal}
 
-## Common restriction reasons and cooldowns {#common-reasons}
-
-## Ideas for a multiplayer match {id=идеи-для-мультиплеера} <!-- alternative syntax for Unicode identifiers -->
+<!-- Alternative form, in case the identifier contains anything except letters, numbers, underscores, and dashes -->
+## Various examples of osu! gameplay {id=osu!-gameplay}
 ```
 
 This feature can also be used for tagging a specific part of the article which doesn't have a heading. Use it sparingly:
 
 ```markdown
 > That's it! You're well on your way to becoming an osu! rhythm champion!
-{#tutorial-quote}
+
+::{#tutorial-quote}::
```

Comment:
both are used, one [here](https://github.com/ppy/osu-wiki/blob/master/news/2022/2022-12-08-mappers-choice-awards.md?plain=1#L53) and one [here](https://github.com/ppy/osu-wiki/blob/master/news/2020/2020-02-28-mappers-report-february.md?plain=1#L140)